plan/sort_test.go

```diff
@@ -320,3 +320,310 @@ func TestSortComplex(t *testing.T) {
 		t.Error(diff)
 	}
 }
+
+func TestSort_PlanShouldBeUpdatedCorrectly(t *testing.T) {
```

Comment:
Blip uses test name schema similar to Go: `TestSoemthing`, i.e. doesn't use underscores (are are pretty rare in Go in general). Also, all things should be updated/whatever correctly, so most of this test name is not actually descriptive/precise.